test/integration/api/forms/forms.js

```diff
@@ -600,25 +602,45 @@ describe('api: /projects/:id/forms (create, read, update)', () => {
                 body.submissions.should.equal(0);
               })))));
 
+      // eslint-disable-next-line indent
     it('should return the correct enketoId', testService((service, container) =>
+        // eslint-disable-next-line indent
       service.login('alice', (asAlice) =>
+          // eslint-disable-next-line indent
         asAlice.post('/v1/projects/1/forms?publish=true')
+            // eslint-disable-next-line indent
           .set('Content-Type', 'application/xml')
+            // eslint-disable-next-line indent
           .send(testData.forms.simple2)
+            // eslint-disable-next-line indent
           .expect(200)
+            // eslint-disable-next-line indent
           .then(() => exhaust(container))
+            // eslint-disable-next-line indent
           .then(() => {
+              // eslint-disable-next-line indent
             global.enketoToken = '::ijklmnop';
+              // eslint-disable-next-line indent
             return asAlice.post('/v1/projects/1/forms/simple2/draft')
+                // eslint-disable-next-line indent
               .expect(200)
+                // eslint-disable-next-line indent
               .then(() => exhaust(container))
+                // eslint-disable-next-line indent
               .then(() => asAlice.get('/v1/projects/1/forms/simple2')
+                  // eslint-disable-next-line indent
                 .set('X-Extended-Metadata', true)
+                  // eslint-disable-next-line indent
                 .expect(200)
+                  // eslint-disable-next-line indent
                 .then(({ body }) => {
+                    // eslint-disable-next-line indent
                   body.enketoId.should.equal('::abcdefgh');
+                    // eslint-disable-next-line indent
                   body.enketoOnceId.should.equal('::::abcdefgh');
+                  // eslint-disable-next-line indent
                 }));
+            // eslint-disable-next-line indent
```

Comment:
There are so many `// eslint-disable-next-line indent` comments inserted at each line here and elsewhere in this diff that i'd love to see the indentation just fixed. (I think it's only this way because no linter was yelling at the author in the first place!) _Or_ have the eslint comment applied to the whole test block or file. Then we can come back and fix the indention later, but not have to remove every one of these new lines.